variables.tf

```diff
@@ -174,7 +174,7 @@ variable "gateway_probe_threshold" {
 
 # TLS
 variable "tls_certificate_path" {
-  description = "Path to a TLS certificate that will be imported in Azure Key Vault and used in the Application Gateway TLS listener for GraphDB. Either tls_certificate_path or tls_certificate_password must be provided."
+  description = "Path to a TLS certificate that will be imported in Azure Key Vault and used in the Application Gateway TLS listener for GraphDB. Either tls_certificate_path or tls_certificate_id must be provided."
```

Comment:
Well this `Either tls_certificate_path or tls_certificate_id must be provided.` part should be included as well in the `tls_certificate_id` variable.